Cubical/Categories/Profunctor/Base.agda

```diff
@@ -0,0 +1,307 @@
+{-
+
+  Definition of profunctors (https://ncatlab.org/nlab/show/profunctor)
+  and some basic facts about them.
+
+  We define a profunctor C -/-> D as a functor C^o x D -> Set. We pick
+  the universe levels such that the hom sets of C and D match Set,
```

Comment:
As mentioned on the nlab, different people follow different conventions. This convention has the benefit of matching how you apply it to arguments. I.e., if R : C -/-> D then the set is R.Het[ c , d ] whereas with the opposite convention it would be R.Het[ d , c ] . The alternative convention has the benefit that the functor `Cat -> Prof` is doesn't involve an op or co, but I don't think that that will matter so much in this library that is for the moment quite concrete.

Reply:
Mike Shulman on Zulip made the following suggestion: we could define both `C ⊶ D` and `D ⊷ C` to mean profunctors where the open side is contravariant (to suggest an "op"). This has the downside that the unicode characters here don't have names in Agda-mode, but I think it makes it nicely unambiguous and doesn't globally fix an orientation.

The only naming issue is what we call the category of all such profunctors, maybe just PROF⊶ C D and  PROF ⊷ C D?

Reply:
I see, thanks for explaining!
Just in case this is not clear: I was just pointing something out I was confused about - I'm fine with accepting the code as is.
I like the suggestion though and I don't mind using characters without  a name in the agda-mode.